networking/ptp/configuring-ptp.adoc

```diff
@@ -73,11 +73,15 @@ include::modules/nw-ptp-configuring-linuxptp-services-as-ordinary-clock.adoc[lev
 
 include::modules/nw-columbiaville-ptp-config-refererence.adoc[leveloffset=+2]
 
+include::modules/nw-ptp-configuring-linuxptp-services-dual-port-oc.adoc[leveloffset=+2]
+
 [role="_additional-resources"]
 .Additional resources
 
 * For a complete example CR that configures `linuxptp` services as an ordinary clock with PTP fast events, see xref:../../networking/ptp/configuring-ptp.adoc#configuring-linuxptp-services-as-ordinary-clock_configuring-ptp[Configuring linuxptp services as ordinary clock].
```

Comment:
Consider dropping the link sentence to keep consistency among the new link style.

Reply:
I'd like to escape changing this as it's outside the changes for this PR. And while it's against style guidelines, it adds to user understanding in this instance.